### PR TITLE
[POI] Replace the carousel of results with a single card

### LINKED ISSUE #2207: [POI] Replace the carousel of results with a single card
#### Is your feature request related to a problem? Please describe. 
The carousel card display for POI results is large and a lot to navigate, this would better be consolidated in a single card

#### What is the solution you are looking for?
image

Return a single PointOfInterestResults card
image

- [ ] Apply bleed to the header
- [ ] Remove the white background
- [ ] Add container with Azure Maps image showing user and POI pins
- [ ] Add container w/ columns for each POI, user can select on one with submit action or navigate through the dialog as it currently works
- [ ] Remove suggested actions when results are displayed (too much at once)

### PR DESCRIPTION
**Pull request by xieofxie**
### Purpose
*What is the context of this pull request? Why is it being done?*

Close #2207 

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

* Use combined card for locations like below
* Remove suggested actions when results are displayed
* Add an alternative address for azure address search for better display in asking current location
* Add routeLimit in appsettings.json

When use foursquare:
f0
f1

When use azure:
a1

When in Teams:
t1

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

Yes.
Note that a test is changed.

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [x] I have commented my code, particularly in hard-to-understand areas	
- [x] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [x] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

**Review by lzc850612 — APPROVED** [review]

File: skills/src/csharp/pointofinterestskill/pointofinterestskill/Content/PointOfInterestDetails.1.0.json [hunk 1]
```diff
@@ -10,102 +9,61 @@
...
+                  "spacing": "None",
+                  "color": "Default",
+                  "text": "{AvailableDetails}"
+                },
+                {
+                  "type": "TextBlock",
+                  "spacing": "None",
+                  "color": "Default",
+                  "text": "{Address}",
+                  "wrap": true
```
Comment (ryanisgrig) [inline skills/src/csharp/pointofinterestskill/pointofinterestskill/Content/PointOfInterestDetails.1.0.json]: Let's try it with "maxLines":"1"  for this text block

File: skills/src/csharp/pointofinterestskill/pointofinterestskill/Content/PointOfInterestDetails.json [hunk 2]
```diff
@@ -1,129 +1,67 @@
...
+                  "type": "TextBlock",
+                  "spacing": "None",
+                  "color": "Dark",
+                  "text": "{AvailableDetails}"
+                },
+                {
+                  "type": "TextBlock",
+                  "spacing": "None",
+                  "color": "Dark",
+                  "text": "{Address}",
```
Comment (ryanisgrig) [inline skills/src/csharp/pointofinterestskill/pointofinterestskill/Content/PointOfInterestDetails.json]: Let's try it with "maxLines":"1" for this textblock

File: skills/src/csharp/pointofinterestskill/pointofinterestskill/Content/PointOfInterestDetails.json [hunk 3]
```diff
@@ -1,129 +1,67 @@
...
-    "type": "Action.Submit",
-    "data": "{SubmitText}"
-  },
   "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
-  "version": "1.0",
-  "speak": "{Speak}"
-}
\ No newline at end of file
+  "version": "1.0"
+}
```
Comment (ryanisgrig) [inline skills/src/csharp/pointofinterestskill/pointofinterestskill/Content/PointOfInterestDetails.json]: Add the ProviderDisplayText container back in, I may have removed it in mock up (sorry!)

File: skills/src/csharp/pointofinterestskill/pointofinterestskill/Content/PointOfInterestDetails.1.0.json [hunk 4]
```diff
@@ -10,102 +9,61 @@
...
-          "color": "default",
-          "text": "{ProviderDisplayText}"
-        }
-      ]
+      ],
+      "selectAction": {
+        "type": "Action.Submit",
+        "data": "{SubmitText}"
+      }
     }
```
Comment (ryanisgrig) [inline skills/src/csharp/pointofinterestskill/pointofinterestskill/Content/PointOfInterestDetails.1.0.json]: Add the ProviderDisplayText container back in

File: skills/src/csharp/pointofinterestskill/pointofinterestskill/Content/PointOfInterestDetailsContainer.json [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+      ]
+    },
+    {
+      "type": "Container",
+      "id":  "Container",
+      "items": [
+      ]
+    }
+  ],
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
```
Comment (ryanisgrig) [inline skills/src/csharp/pointofinterestskill/pointofinterestskill/Content/PointOfInterestDetailsContainer.json]: Add the ProviderDisplayText container back in

**Review by ryanisgrig — CHANGES_REQUESTED** [review]

**Comment by ryanisgrig** [comment]
Any idea what the dimensions of the Azure Maps image are? I don't see where it is set in the code.
We should try with the height of the map picture 50-75% of what it is now. I don't think that requires that much space.

**Comment by ryanisgrig** [comment]
Also confirming, does this remove the suggested actions?

**Comment by xieofxie** [comment]
* Add ProviderDisplayText in the Container
* Size is in the AzureMapsGeoSpatialService. Now width and height are parameters and they are configured in PointOfInterestDialogBase.ImageSize
* Prompt is created with Style = ListStyle.None, so no suggested actions now

New styles:
Foursquare:
f0
f1
Azure:
a1
Teams:
t1

**Review by ryanisgrig — APPROVED** [review]

**Pull request merged**